.github/workflows/ci.yaml

```diff
@@ -0,0 +1,27 @@
+name: Continuous integration
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Go 1.14
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.14
+      id: go
+
+    - name: Setup opentelemetry-collector-builder
+      run: |
+        curl -sLo /bin/opentelemetry-collector-builder https://github.com/observatorium/opentelemetry-collector-builder/releases/download/v0.1.0/opentelemetry-collector-builder_0.1.0_linux_x86_64
```

Comment:
Just wondering if the version of the builder could be obtained from a config at the root of the repo - so when new versions are being updated, we don't need to dig out any location where the version is being used?

Reply:
Absolutely, this is just a first version to get things in place. I'm also not quite happy that I have most of ci and release duplicate. Once everything is working, I'll clean it up a bit.